addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3846,6 +3850,34 @@ export class OdooEditor extends EventTarget {
                             setSelection(codeElement.firstChild, 0);
                         }
                     }
+                } else if (
+                    ev.data === ' ' &&
+                    shouldCreateNumberList &&
+                    // It should not create list with "1.  " or "a.  " etc..
+                    shouldCreateList &&
+                    !closestElement(selection.anchorNode, 'li')
+                ) {
+                    if (stringToConvert.startsWith('A')) {
+                        closestBlock(selection.anchorNode).style.listStyle = 'upper-alpha';
+                    } else if (stringToConvert.startsWith('a')) {
+                        closestBlock(selection.anchorNode).style.listStyle = 'lower-alpha';
+                    }
```

Comment:
We shouldn't apply these styles here until it's converted to a list. At this stage, there's no list available. The code should be moved below once we have a list.